docker/pre-install

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Modified version of github.com/spdk/docker
+
+set -e
+
+spdk_repo=/tmp/spdk
+
+cleanup() {
+
+	rm -f "$HOME/rpmbuild/rpm/x86_64/"*.rpm
+	rm -f "$spdk_tar"
+}
+
+trap 'cleanup' EXIT
+
+# Spice it a bit with supported sources
+"$spdk_repo/scripts/pkgdep.sh" -d
+
+# HACK: In case we received a .tar with built SPDK we need to overwrite the
+# configuration to update all the paths make would need to lookup - this is
+# needed since we execute inside a different mount namespace so we won't be
+# able to find any absolute paths that were used prior creating the .tar.
+
+"$spdk_repo/configure" --with-rbd > /dev/null
+
+# Deploy SPDK inside the container
+
+SPDK_VERSION="${spdk_version}" DEPS="no" "$spdk_repo/rpmbuild/rpm.sh"
```

Comment:
And the rpm build also needs tweaking for building with rbd support. I had this patch [1] that added `--with-rbd` option.

[1] https://review.spdk.io/gerrit/c/spdk/spdk/+/11135

Reply:
@trociny I see both had a CI failure (but I cannot access logs to check what's that about) and also need a rebase. Could you please address that? I added this as a topic for tomorrow's meeting agenda (https://pad.ceph.com/p/rbd_nvmeof). Thank you!

Reply:
@epuertat Rebased.  I have no idea about CI failure. The last time I looked at it it did not look like related to my change.

Reply:
Thank you, @trociny! I see [one of your patches](https://review.spdk.io/gerrit/c/spdk/spdk/+/11135) is ready for "merging" (not familiar with Gerrit terminology), while [the other](https://review.spdk.io/gerrit/c/spdk/spdk/+/11134/) had some CI failure and Tomasz needs to validate your replies.

Reply:
@trociny there's still one comment unaddressed here: https://review.spdk.io/gerrit/c/spdk/spdk/+/11135/1..6//COMMIT_MSG#b8

Let's hope that @sdpeters can merge those 2 changesets after that.